zubhub_backend/zubhub/creators/utils.py

```diff
@@ -456,3 +458,90 @@ def send_notification(users: List[Creator], source: Creator, contexts,
 #         EmailAddress.objects.create(
 #             user=user, email=email, primary=False, verified=False
 #         )
+
+def set_badge_view_category(creator):
+    views_count= Project.objects.filter(creator= creator).aggregate(Sum(('views_count')))["views_count__sum"]
+
+    List = ["Getting Famous", "Person of Interest", "Popular Projects", "Idea Factory"]
+
+    creator.badges.remove(Badge.objects.get(badge_title = List[3]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[2]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[1]))
+    creator.badges.remove(Badge.objects.get(badge_title = List[0]))
+    
+    if(views_count > 100000):
```

Comment:
can we move these constants to a hashmap. If this logic is used at multiple places it'll be helpful, or otherwise as well.
something like
`getting_famous: 10000` and pick the 10000 from this map.